release_bot/cli.py

```diff
@@ -31,6 +31,8 @@ def parse_arguments():
                             default='')
         parser.add_argument("-v", "--version", help="display program version", action='version',
                             version=f"%(prog)s {configuration.version}")
+        parser.add_argument("-r", "--dryrun", help="Perfroms no remote changes.",
```

Comment:
my role model is `git`, can we rename the short opt to `-n`, exactly as git has it? From git-clean manpage:
```
       -n, --dry-run
           Don’t actually remove anything, just show what would be done.
```

Reply:
I also misspelled `Perfroms`. Stupid me. :P

Done!